### DISCUSSION TITLE
[@types/express-serve-static-core] TS2339: Property 'status' does not exist on type 'Response<any, Record<string, any>>

### DISCUSSION THREAD

**Discussion by ensgenc** [DefinitelyTyped/DefinitelyTyped#62263] [Issues with a @types package]
Hi,
"@types/express-serve-static-core": "4.17.28"  there is no problem in this version but there is a type error in the new version.

Node: v12
express: "4.17.1"

Error: 
TS2339: Property 'method' does not exist on type 'Request'.
TS2339: Property 'body' does not exist on type 'Request'.
TS2339: Property 'status' does not exist on type 'Response<any>'.
TS2339: Property 'headers' does not exist on type 'Request<ParamsDictionary, any, any, ParsedQs, Record<string, any>>'.
TS2339: Property 'send' does not exist on type 'Response<any>'.
TS2339: Property 'originalUrl' does not exist on type 'Request'.
TS2339: Property 'query' does not exist on type 'Request'.
TS2339: Property 'ip' does not exist on type 'Request'.

We started taking it yesterday. Would you check.

Thanks

[a comment by an automated account is not shown]

**Comment by AlexisN94** [comment]
I'm experiencing the same issue. For now I've reverted to version 4.17.28, but I'd expect a fix in the meantime
